bevy-jornet/examples/whac-a-square.rs

```diff
@@ -230,7 +230,9 @@ mod menu {
             if let Some(player) = leaderboard.get_player() {
                 player_name.single_mut().sections[1].value = player.name.clone();
             }
-            let leaderboard = leaderboard.get_leaderboard();
+            let mut leaderboard = leaderboard.get_leaderboard();
+            leaderboard.sort_unstable_by(|s1, s2| s2.score.partial_cmp(&s1.score).unwrap());
```

Comment:
This changed from https://github.com/vleue/jornet/commit/2ee9e66afd2645cc0cef746ed93c8c56030bfb1a wasn't visible when I rebased 
```suggestion
            leaderboard.sort_unstable_by(|s1, s2| s2.score.partial_cmp(&s1.score).unwrap_or(Ordering::Equal));
```